test/e2e/es_index_cleaner_test.go

```diff
@@ -1,3 +1,5 @@
+// +build elasticsearch
+
```

Comment:
Wondering whether the es-index-cleaner test should just be included in the elasticsearch_suite?